css/reference/ref-filled-green-200px-square.xht

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Please use `.html` for this file, I believe all new things should use HTML5 and not XHTML anymore.